src/base/utils/CoreModel.jl

```diff
@@ -17,3 +17,77 @@ Base.isequal(model1::CoreModelCoupled, model2::CoreModelCoupled) =
     isequal(model1.cu, model2.cu)
 
 Base.copy(model::CoreModelCoupled) = CoreModelCoupled(model.lm, model.C, model.cl, model.cu)
+
+"""
+    find_exchange_reactions(
+        model::CoreModel;
+        exclude_biomass = false,
+        biomass_strings = _constants.biomass_strings,
+        ex_prefixes = _constants.exchange_prefixes,
+    )::Vector{String}
+
+Returns indices of exchange reactions. Exchange reactions are identified based
+on most commonly used prefixes, these prefixes can be set using `ex_prefixes`.
+See `_constants.biomass_strings` for the default list. The biomass reaction is
+also added if it can be identified by looking for the strings listed in
+`biomass_strings` in the reaction ids. If `exclude_biomass` is true then this
+does not occur.
+
+Note: biomass exchange reactions are counted as exchange reactions and will NOT
```

Comment:
So my original intention was to exclude a biomass exchange (and not pickup a biomass reaction in any case). Any reason to change this?

Reply:
hmm not really, fixing now